swan-lake/learn-the-platform/source-code-dependencies/organize-ballerina-code.md

```diff
@@ -40,6 +40,8 @@ $ bal new hello_world
 
 This creates a new Ballerina package  in the default module with the `Ballerina.toml` file, which identifies a directory as a package and a sample source file (i.e., `main.bal`) with a main function.
 
+>**Info:** You can provide a directory path with `bal new` to create a package in a specific directory of your choice. E.g., `bal new <package-path>`.
```

Comment:
IMO we don't need this addition here. The documentation is about creating the first package. Having the help text will suffice.